hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx

```diff
@@ -27,6 +27,8 @@ import { routesV2 } from '@/v2/routes-v2';
 import { MakeRouteWithSubRoutes } from '@/makeRouteWithSubRoutes';
 import classNames from 'classnames';
 
+import Loader from '@/v2/components/loader/loader';
+
 import './app.less';
 
```

Comment:
I noticed that when we click on "Show ACL," the sidebar displaying the various ACLs associated with the volume appears, which is great. However, after clicking "Show ACL" and then closing it, if we try to add or remove a column from the table, the ACL sidebar keeps popping up repeatedly. Could you please fix this issue?

Reply:
Thanks for pointing this out @ArafatKhan2198 ,
fixed it in [40c2588](https://github.com/apache/ozone/pull/7048/commits/40c2588d021292463b1012006975ed2d33c33c22)